packages/integration-tests/src/collections/CustomReference.test.ts

```diff
@@ -1,66 +1,74 @@
 import { EntityManager } from "joist-orm";
-import { knex, numberOfQueries, resetQueryCount } from "../setupDbTests";
-import { Author, Book, BookReview } from "../entities";
-import { insertAuthor, insertBook, insertBookReview, insertPublisher } from "../entities/factories";
+import { Author, BookReview, Image, ImageType } from "../entities";
+import { insertAuthor, insertBook, insertBookReview, insertImage } from "../entities/factories";
+import { knex } from "../setupDbTests";
 
 describe("CustomReference", () => {
```

Comment:
I moved this one to use the new `Image` b/c `BookReview` is using `hasOneThrough` now, so the `hasOneThrough.test.ts` covers those code paths, and the code paths these tests were for are now in the `Image.owner`.